jena-arq/src/main/java/org/apache/jena/sparql/engine/JsonIterator.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.jena.sparql.engine;
+
+import java.util.Collections;
+import java.util.Iterator;
+import java.util.List;
+import java.util.NoSuchElementException;
+
+import org.apache.jena.atlas.json.JsonObject;
+import org.apache.jena.atlas.json.JsonValue;
+import org.apache.jena.atlas.lib.Closeable;
+
+import org.apache.jena.graph.Node;
+import org.apache.jena.sparql.core.Var;
+import org.apache.jena.sparql.engine.binding.Binding;
+import org.apache.jena.sparql.lib.RDFTerm2Json;
+
+/** A JSON iterator for JsonObject's, that wraps a QueryIterator, and a list
+ * of result variables.
+ */
+public class JsonIterator implements Iterator<JsonObject>
+{
```

Comment:
Feel free to reformat int same like `{` and no space before `;`.

(yes - a lot of code is not formatted but we seem to be heading that way).

Or leave for now and we can clean up new classes after merging.

Reply:
I've been planning to offer a low-interruption plan for some large-scale reformatting later this spring, so no harm in leaving this.

Reply:
Okeydokey @afs I noticed new classes are getting the new formatting applied, but wasn't sure about changing it in this PR or leaving as it was. This change started somewhere in 2015, so lots of old code formatting here.